pkg/registry/graph.go

```diff
@@ -0,0 +1,22 @@
+package registry
+
+type Package struct {
+	Name           string
+	DefaultChannel string
+	Channels       map[string]Channel
+}
+
+type Channel struct {
+	Head     BundleKey
+	Replaces map[BundleKey]map[BundleKey]struct{}
```

Comment:
What happened here? Cna we define this? like `ReplaceMap`?

Reply:
What about `nodes` ?